Sources/Sentry/Public/SentrySDK.h

```diff
@@ -382,6 +382,19 @@ SENTRY_NO_INIT
  */
 + (void)startProfiler;
 
+/**
+ * Start a new continuous profile session.
+ * @note Does nothing if the profiling session is sampled with respect
+ * @c SentryOptions.profileSessionSampleRate and the profiler is already running.
+ * @note Does nothing if the profile session is not sampled.
+ * @note Does nothing if @c SentryOptions.profileLifecycle is set to @c trace . In this scenario,
+ * the profiler is automatically started and stopped depending on whether there is an active sampled
+ * span so it is not permitted to manually start profiling.
+ * @warning Continuous profiling mode is experimental and may still contain bugs.
```

Comment:
`m`: Maybe we should put the warning above all the `@note` so it's clearer this is still experimental.

Reply:
Done!